api/src/main/java/jakarta/data/metamodel/ComparableAttribute.java

```diff
@@ -29,25 +29,23 @@
  * <p>Entity attribute types that are comparable include:</p>
  *
  * <ul>
- * <li>numeric attributes, such as {@code long}, {@link Float}, and
- *     {@link java.math.BigInteger}</li>
  * <li>time attributes, such as {@link java.time.LocalDateTime} and
  *     {@link java.time.Instant}</li>
  * <li>boolean attributes: {@code boolean} and {@link Boolean}</li>
  * <li>enum attributes - Note that it is provider-specific whether order is
  *     based on {@link Enum#ordinal()} or {@link Enum#name()}.
  *     The Jakarta Persistence default of {@code ordinal} can be overridden
  *     with the {@code jakarta.persistence.Enumerated} annotation.</li>
+ * <li>numeric attributes, such as {@code long}, {@link Float}, and
+ *     {@link java.math.BigInteger} - Use the {@link NumericAttribute} subtype
+ *     instead</li>
  * <li>textual attributes - Use the {@link TextAttribute} subtype instead</li>
```

Comment:
Something weird going on with the punctuation of this bullet list.

Reply:
> Something weird going on with the punctuation of this bullet list.

You are right, but I expect this list to be redone after we split out the temporal types and possibly boolean to their own subtypes of Attribute, so it might not be worth it to try to clean it up yet.

Reply:
Yeah no worries.